user_model.ml

```diff
@@ -1,10 +1,13 @@
 module Rng = Mirage_crypto_rng
 
 type token = {
+  name : string;
   token_type : string;
```

Comment:
the token type is always "Bearer"?

Reply:
At the moment yes, it's always `Bearer` but I like that it's flexible and we can always change the authentication type. maybe `Basic` someday?